apm-sniffer/apm-sdk-plugin/dubbo-2.7.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/asf/dubbo/DubboInterceptor.java

```diff
@@ -33,6 +34,7 @@
 import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.InstanceMethodsAroundInterceptor;
 import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.MethodInterceptResult;
 import org.apache.skywalking.apm.network.trace.component.ComponentsDefine;
+import org.springframework.util.StringUtils;
```

Comment:
Don't important `Spring.util`. We have provided `org.apache.skywalking.apm.util.StringUtil` at the agent core level

Reply:
done